friendshipper/src/lib/components/playtests/PlaytestModal.svelte

```diff
@@ -256,7 +262,7 @@
 					size="sm"
 					name="minGroups"
 					min="1"
-					max="10"
+					max={$dynamicConfig.playtestDiscordChannels.length + 1}
```

Comment:
i feel like maybe we either set max to a higher const or remove it altogether - something irks me about requiring discord to playtest at all, yknow?

Reply:
I agree, I kinda hated doing it. 

In an ideal world I'd love some reasonable triggerpoint with a warning, requiring a "Confirm" to continue anyway. (Just to underscore the resource allocation it's making.) 

I think just bumping to, say, 25 would be fine, too.